### PR TITLE
Re-enable Rider debug tests on 2020.3

### PR DESCRIPTION
**Pull request by abrooksv**
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-rider/it/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotnetLocalLambdaRunConfigurationIntegrationTest.kt [hunk 1]
```diff
@@ -120,13 +123,24 @@ abstract class DotnetLocalLambdaImageRunConfigurationIntegrationTestBase(private
     private val mockId = "MockCredsId"
     private val mockCreds = AwsBasicCredentials.create("Access", "ItsASecret")
 
+    private var initialImmediateWindow: Boolean = false
```
Comment (hunterwerlla) [inline jetbrains-rider/it/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotnetLocalLambdaRunConfigurationIntegrationTest.kt:126]: Can you add a comment on why

Comment (abrooksv) [reply jetbrains-rider/it/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotnetLocalLambdaRunConfigurationIntegrationTest.kt:126]: Sure, i can move it. I put it into the setuo methdod

**Review by hunterwerlla — APPROVED** [review]

**Pull request merged**